qdb/memqdb.go

```diff
@@ -378,6 +378,16 @@ func (q *MemQDB) DeleteRouter(ctx context.Context, id string) error {
 	return ExecuteCommands(q.DumpState, NewDeleteCommand(q.Routers, id))
 }
 
+func (q *MemQDB) OpenRouter(ctx context.Context, id string) error {
+	spqrlog.Zero.Debug().Str("router", id).Msg("memqdb: open router")
```

Comment:
```
spqrlog.Zero.Debug().
    Str("router", id).
    Msg("memqdb: open router")
```
</stupid_review_advices>

And add some feature tests for opening/closing router please